frontend/src/Onboarding/OnboardingInfoScreen.tsx

```diff
@@ -143,7 +138,7 @@ class OnboardingScreenComponent extends React.Component<
 
   onChangeGradYear(e: any) {
     this.setState({
-      gradYear: Number(e.target.value),
+      gradYear: e.target.value == 2020 ? Number(e.target.value) : 2020,
```

Comment:
What's the reason for this?

Reply:
Oh shoot I was trying to test something and forgot to remove that